instrumentation/apolloconfig/apolloconfig-2.0.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apolloconfig/v2_0_0/ApolloConfigInstrumentationModule.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apolloconfig.v2_0_0;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.Collections;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class ApolloConfigInstrumentationModule extends InstrumentationModule {
+  public ApolloConfigInstrumentationModule() {
+    super("apolloconfig", "apolloconfig-2.0.0");
+  }
+
+  @Override
+  public List<TypeInstrumentation> typeInstrumentations() {
+    return Collections.singletonList(new ApolloRepositoryChangeInstrumentation());
```

Comment:
We usually use static import here, for detail, you can refer to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports